charts/apps/kubeflow-pipelines/s3-only-static/values.yaml

```diff
@@ -1,4 +1,4 @@
 s3:
   bucketName: 
   minioServiceHost: s3.amazonaws.com
-  minioServiceRegion:
+  minioServiceRegion: 
```

Comment:
revert?

Reply:
It should be whatever describe from here when running helmify https://github.com/jsitu777/kubeflow-manifests/blob/main/tools/helmify/template/kubeflow-pipelines/values/s3-only-static/values.yaml

Reply:
got it, can we remove whitespaces from the templates?

Reply:
removed